src/sqlstore/match.py

```diff
@@ -0,0 +1,37 @@
+
+from sqlalchemy import Table, Column, Integer, String, BigInteger, Boolean, ForeignKeyConstraint, Numeric, DateTime
+from sqlalchemy.sql import func
+from src.sqlstore.db import Base
+
+
+class SQLmatch(Base):
+    __tablename__ = "match"
+    matchId = Column("matchId", String(20), primary_key=True)
+    platformId = Column("platformId", String(7))
+    gameId = Column("gameId", BigInteger)
+    seasonId = Column("seasonId", Integer)
+    patch = Column(Integer)
+    queueId = Column("queueId", Integer)
+    gameVersion = Column("gameVersion", String(23))
+    mapId = Column("mapId", Integer)
```

Comment:
Can but not must. Normally ID are stored in Strings for flexibility. But not really neccessary.